.agent/rules/ponder-multi-chain-config.md

```
@@ -0,0 +1,379 @@
+    // Do mainnet-specific stuff!
+  }
+});
+```
+
+#### Chain override logic
+
+Chain-specific configuration uses an override pattern. Any options defined at the top level are the default, and the chain-specific objects override those defaults.
+
+All contract options other than `abi` can be specified per-chain, including `address`, `startBlock`, and `endBlock`.
+
+**Example: Uniswap V3**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)